release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2730,6 +2730,45 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.19 --pullspecs
+----
+
+[id="ocp-4-15-19-bug-fixes"]
+==== Bug fixes
+
+* Previously, when a new version of a custom resource definitions (CRD) specifies a conversion strategy, the assumption was that the new conversion strategy would successfully convert resources. This was not the case because Operator Lifecycle Manager (OLM) cannot run the new conversion strategies for Custom Resource Definition (CRD) validation without actually performing the upgrade operation. OLM cannot guarantee that an existing conversion strategy converts the same as the specified strategy. With this release, the OLM implementation generates a warning message during the upgrade process when CRD validations fail with the existing conversion strategy and the new conversion strategy is specified in the new version of the CRD. (link:https://issues.redhat.com/browse/OCPBUGS-35720[*OCPBUGS-35720*]).
+
+* Previously, during node reboots, especially during upgrade operations, the node that interacts with the rebooting machine enters a `Ready=Unknown` state for a short amount of time. This situation caused the Control Plane Machine Set Operator to enter an `UnavailableReplicas` condition and then an `Available=false` state. The `Available=false` state triggers alerts that demand urgent action, but intervention would only be required for a short period until the node reboots. With this release, a grace period for node unreadiness is provided where if a node enters an unready state, the Control Plane Machine Set Operator will not instantly enter an `UnavailableReplicas` condition or an `Available=false` state. (link:https://issues.redhat.com/browse/OCPBUGS-34971[*OCPBUGS-34971*]).
+
+* Previously, the OpenShift Cluster Manager container did not have the right TLS Certificates. As a result, image streams could not be used in disconnected deployments. With this update, the TLS Certificates are added as projected volumes. (link:https://issues.redhat.com/browse/OCPBUGS-34580[*OCPBUGS-34580*])
+
+* Previously, when a serverless function was created in the create serverless form, BuilldConfig was not created. With this update, if the Pipelines Operator is not installed, if the pipeline resource is not created for particular resource, or if the pipeline is not added while creating a serverless function, it will create BuildConfig as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34350[*OCPBUGS-34350*])
+
+* Previously, the reduction of the network queue did not work as expected for inverted rules such as `!ens0`. This happened because the exclamation mark symbol was duplicated in the generated tuned profile. With this release, the duplication no longer occurs so that inverted rules would as intended. (link:https://issues.redhat.com/browse/OCPBUGS-33929[*OCPBUGS-33929*]).
```

Comment:
```suggestion
* Previously, the reduction of the network queue did not work as expected for inverted rules such as `!ens0`. This happened because the exclamation mark symbol was duplicated in the generated tuned profile. With this release, the duplication no longer occurs so that inverted rules apply as intended. (link:https://issues.redhat.com/browse/OCPBUGS-33929[*OCPBUGS-33929*]).
```

Reply:
:heavy_check_mark: